src/gql/generated/types.ts

```diff
@@ -1,1233 +0,0 @@
-export type Maybe<T> = T | null;
```

Comment:
I think you accidentally deleted this file

Reply:
indeed. should be back now